linting.py

```diff
@@ -1,20 +1,13 @@
 # -*- coding: utf-8 -*-
-import re
-import sys
-import argparse
+import re, sys, argparse
 from pylint import lint
 
+# Call `echo $?` to see the exit code after a run. If the score is over this, the exit
+# code will be 0 (success), and if not will be nonzero (failure).
 THRESHOLD = 8.5
 
 if len(sys.argv) < 2:
     raise argparse.ArgumentError("Module to evaluate needs to be the first argument")
 
 sys.argv[1] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[1])
-run = lint.Run([sys.argv[1]], do_exit=False)
-score = run.linter.stats['global_note']
-
-if score < THRESHOLD:
-    print("Your code doesn't pass the PEP8 style score threshold: %f!" % THRESHOLD)
-    sys.exit(1)
-
-print("Congratulations! Your code has passed the PEP8 style score threshold: %f!" % THRESHOLD)
\ No newline at end of file
+run = lint.Run([sys.argv[1], f"--fail-under={THRESHOLD}"])
```

Comment:
I was having some trouble because the `do_exit` parameter no longer exists, so `Run` always does an `exit` call, which means anything below it doesn't get run. There are ways to catch that exit and continue with a script, but it turns out you can pass the threshold to the `Run`, then the exit code returned by `python3 linting.py your_file.py` is 0 or nonzero as appropriate. I didn't think the last congratulatory or failure message was actually necessary, since if you're running this script you're probably aware of the threshold and can read a `pylint` printout in all its verbose glory.